src/avfoundation.cs

```diff
@@ -10991,13 +10991,15 @@ interface AVPlayerLooper
 		[DesignatedInitializer]
 		IntPtr Constructor (AVQueuePlayer player, AVPlayerItem itemToLoop, CMTimeRange loopRange);
 
+#if !XAMCORE_4_0 // This API got introduced in Xcode 8.0 binding but is not currently present nor in Xcode 8.3 or Xcode 9.0 needs research
+		[Internal]
+		[Export ("disableLooping")]
+		void _DisableLooping ();
+#else
```

Comment:
If I follow the logic of this patch: "API got introduced in Xcode 8.0 binding but is not currently present" then I don't see why DisableLooping should remain in the else clause. Nothing tells use that if we ever go XAMCORE_4_0 this API will be there (and working).

Reply:
No no. What is out is the property. The method, DisableLooping is present and is correct. The issue here is as follows.

* Looper always loops until told otherwise. 
* LoopingEnabled should return true by default.
* If DisableLooping is called, we should correctly return false.

Then, in XAMCORE_4_0 we want to have a DisableLooping binding that does not do the setting of an unnecessary internal variable. Therefore, we have the #else to provide an automatic binding (and no need for tests).

Reply:
That confused me too the first time I read it. Let's simplify this with

```
#if !XAMCORE_4_0
[PostSnippet ("loopingEnabled = false;")]
#endif
[Export ("disableLooping")]
void DisableLooping ();
```

Reply:
Also please remove the `// This API got introduced in Xcode 8.0 binding but is not currently present nor in Xcode 8.3 or Xcode 9.0 needs research` and maybe annotate Sebastien's `[PostSnippet ("loopingEnabled = false;")]`